### DISCUSSION TITLE
How to transfer an already owned domain onto a Github site?

### DISCUSSION THREAD

**Discussion by Pteronaps** [community/community#138854] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello! Not sure what category to add this to. I want to add a domain name that I already own to my Github site. The only issue is that the name is currently being used for a Wix website that I own. I want to transfer the domain the Wix website has to my Github one, and I know that Wix has a transfer system, but I am unsure on how I'd get the name onto Github..

If anyone knows how to help, that would be great! Thank you!

**Comment by phuyalgaurav** [comment]
where did you buy the domain from?
You have to register the DNS records in your control panel (where you bought the domain)
I suggest looking at this page for more info: [Setting Up Github Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-a-subdomain).

**Reply by Pteronaps** [reply]
It's connected to Wix. I am able to get a code to transfer the domain off of Wix, I just don't know where in Github I'd have to input that.

**Reply by phuyalgaurav** [reply]
did you buy the domain in wix?

**Reply by Pteronaps** [reply]
Yes. Wix allows you to transfer domains that you buy using them off of their service, and I am having trouble trying to get the domain to go into the Github site and off of Wix itself.

**Reply by phuyalgaurav** [reply]
i think you re trying to point the A domain (example.com not somemthing.example.com) so here's what you need to do.

* Create "A records" in your panel add add these ip's
```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```
>for me ( you may have different ui), it looks something like this:
![image](https://github.com/user-attachments/assets/7dd1035e-a557-4a87-9242-4e90de4b3985)

* Now register your domain in github
>Example
![image](https://github.com/user-attachments/assets/078b315c-a336-4888-be46-204c50c4bd85)


You should be all good. If you have any problem please share screenshots :D

**Reply by phuyalgaurav** [reply]
If you did it mark this as an answer.

**Reply by Pteronaps** [reply]
![help](https://github.com/user-attachments/assets/51c77164-2e1d-45f8-8e07-f2b63adb48a2)

This is my current issue! I own the www.pteronaps.com domain as an FYI.

**Reply by phuyalgaurav** [reply]
Did you do step no.1 creating "A records" in your dns panel

**Reply by Pteronaps** [reply]
Sorry for the late response! Yes I did, I added the four. The https://pteronaps.com/ link is working, but not www.pteronaps.com . I am unsure why the www. isn't working? It is a little annoying to have to input the https every single time.
![help 2](https://github.com/user-attachments/assets/4bfaab0e-7c45-4fc1-a7c8-36773e483918)

**Reply by phuyalgaurav** [reply, marked as answer]
Hmm... that should've worked for both. Try adding the www.petronaps.com in the github custom domain.

**Reply by Pteronaps** [reply]
Only the www.pteronaps.com link is working now, but that's fine! That's the link I wanted to work! Thank you for your help ^^!!

**Reply by phuyalgaurav** [reply]
No problem mate!
